controller/controller_test.go

```diff
@@ -470,3 +530,166 @@ func TestController_Status_ErrorOnMismatchingSliceIDs(t *testing.T) {
 	Expect(status).To(BeEmpty())
 	mock.AssertExpectationsForObjects(t, fleetMock.Mock)
 }
+
+func TestController_WaitForStatus_001(t *testing.T) {
```

Comment:
`I_Don't_Like_These_Kind_Of_Test_Names_Because_Even_Making_It_Longer_Questions_How_Good_The_Hint_Really_Is`. What about simply documenting the necessary information with some comments? Then you have a nice description, a proper hint, and the method name stays nice and at least kind of short.